applications/vessel-history/src/types/index.ts

```diff
@@ -190,4 +136,6 @@ export type GFWDetail = {
   shipname: string
   source: string
   dataset: string
+  geartype: string
+  normalized_shipname: string
```

Comment:
If we are using camelCase everywhere, why is this snake_case ?

Reply:
To match the vessel API endpoint response 🤷‍♂️ , for some reason this field is returned with a different form.